File: Data Structures & Algorithms/k-closest-points-to-origin/submission-1.py
class Solution:
    def kClosest(self, points: List[List[int]], k: int) -> List[List[int]]:
        maxHeap = []
        for x, y in points:
            dist = x ** 2 + y ** 2
            heapq.heappush(maxHeap, [dist, x, y])

        res = []
        for _ in range(k):
            dist, x, y = heapq.heappop(maxHeap)
            res.append([x, y])
        
        return res

        

File: Data Structures & Algorithms/reverse-bits/submission-1.rs
impl Solution {
    pub fn reverse_bits(mut n: u32) -> u32 {
        let mut res = 0;
        for _ in 0..32 {
            let bit = n & 1;
            res = (res << 1) | bit;
            n >>= 1
        }
        res
    }
}
